sql/hive/src/test/scala/org/apache/spark/sql/parquet/parquetSuites.scala

```diff
@@ -263,6 +356,22 @@ abstract class ParquetTest extends QueryTest with BeforeAndAfterAll {
     }
   }
 
+  Seq("partitioned_parquet_with_key_and_complextypes", "partitioned_parquet_with_complextypes").foreach { table =>
```

Comment:
For indentation and whitespace related styling, please refer to equivalent changes in #4792.